easybuild/easyblocks/p/python.py

```diff
@@ -156,6 +156,19 @@ def auto_detect_lto_support(self):
 
     def configure_step(self):
         """Set extra configure options."""
+
+        # Check for and report distutils user configs which may make the installation fail
+        # See https://github.com/easybuilders/easybuild-easyconfigs/issues/11009
+        for cfg in [os.path.join(os.path.expanduser('~'), name) for name in ('.pydistutils.cfg', 'pydistutils.cfg')]:
+            if os.path.exists(cfg):
+                self.log.debug('Found distutils user config file at %s', cfg)
+                contents = read_file(cfg)
+                if any(line.startswith('prefix=') for line in contents.splitlines()):
+                    print_warning('Distutils user config found at %s containing a "prefix=" setting. '
+                                  'Installation might fail due to Python picking up that prefix. '
+                                  'Remove the file if this happens!',
+                                  log=self.log)
```

Comment:
AFAIK `pydistutils` has been deprecated for a while in favour of virtual environments. Therefore, it might safer to just error out if its presence can cause issues.
```suggestion
                raise EasyBuildError("Legacy distutils user configuration file found at %s. Aborting.", cfg)
```

Reply:
I didn't want to be too restrictive. But sure that is easier